spirv_msl.cpp

```diff
@@ -10682,68 +11957,529 @@ void CompilerMSL::OpCodePreprocessor::check_resource_write(uint32_t var_id)
 		uses_resource_write = true;
 }
 
+// Storage buffer robustness
+// Fix loads from tessellation control inputs not being forwarded to the gl_in structure array
+// Fix loads from tessellation evaluation inputs not being forwarded to the stage_in structure array
+std::string CompilerMSL::access_chain_internal(uint32_t base, const uint32_t *indices, uint32_t count, AccessChainFlags flags, AccessChainMeta *meta)
```

Comment:
Completely duplicating the function seems a bit overkill.

Reply:
I replaced that by overriding only the [`append_index` function](https://github.com/KhronosGroup/SPIRV-Cross/pull/1162/files#diff-6a4029d3c45d50818e6f80e915867c64R11738).